dcm4che-net-audit/pom.xml

```diff
@@ -40,5 +40,15 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+       <groupId>org.hamcrest</groupId>
+       <artifactId>hamcrest-library</artifactId>
```

Comment:
Hamcrest usually goes before junit because junit library has subset of hamcrest matchers and they collide with each other in Eclipse IDE. Also typically we use "hamrecrest-all" jar.